{{cookiecutter.project_slug}}/frontend/.husky/pre-commit

```diff
@@ -2,5 +2,5 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 # Use the version below while working on cookiecutter itself
-# cd ./{{cookiecutter.project_slug}}/frontend && npx lint-staged
-cd ./frontend && npx lint-staged
+cd ./{{cookiecutter.project_slug}}/frontend && npx lint-staged
+# cd ./frontend && npx lint-staged
```

Comment:
Any reason to keep this line if it is commented?

Reply:
While working on the cookiecutter we need the commented version. But when we create a project with cookiecutter it should have the other version. So i just wanted to to keep commented version in there as well to make it more clear while working on cookiecutter